xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -754,38 +755,44 @@ void CMusicInfoScanner::FileItemsToAlbums(CFileItemList& items, VECALBUMS& album
      */
     for (auto& j : artists)
     {
-      /*
-       Find the common artist(s) for these songs. Take from albumartist tag when present, or use artist tag.
-       When from albumartist tag also check albumartistsort tag and take first non-empty value
+      /* Find the common artist(s) for these songs (grouped under primary artist). 
```

Comment:
remove the white space at the end of the line

Reply:
Sadly Clang plugin on VS2017 does not find trailing white spaces in comments, and such a vital part of the code standard making it so unreadble too.